### DISCUSSION TITLE
Auto scrolling of primary window.

### DISCUSSION THREAD

**Discussion by Manumit32** [vincode-io/Zavala#137] [Support]
I am enjoying the responsiveness and simplicity of Zavala on Mac OS 11.6.  Using Zavala 1.21  for creative brainstorming and exporting to mindmaps and Scrivener so the OPML add-on is excellent.

When in full-screen mode, is there a way to enable/provide automated scrolling when entering data as manual scrolling is proving disruptive?

R.

**Comment by vincode-io** [comment]
Yes, this has been improved in the upcoming 2.0 release of Zavala.
